lib/batch/institutional_nucats_template.rb

```diff
@@ -0,0 +1,40 @@
+# https://github.com/galterlibrary/digital-repository/issues/691
+# Create 10 'template' institutional collections, adding another 3 template
+# collections to each. For more immediate use when needed.
+
+# NUCATS Grants Repository
+parent_nucats_grants = Collection.find("f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d")
+
+10.times do |i|
+  template = Collection.new(
+    title: "template-#{i}", 
```

Comment:
How about also using a timestamp in the title? This way even if you re-run it, you will get templates with different titles for sure (for human convenience).